### ISSUE TITLE
VirtualCam 2.0.4 gets installed under a directory different from that of OBS Studio v.24 x64

### ISSUE THREAD

**Issue by OleksiyRudenko** [CatxFish/obs-virtual-cam#73]
In my particular situation I had OBS Studio v.24.0.3 x64 installed under `/Program Files/obs-studio` (following the default destination suggested by OBS Studio installer) while VirtualCam plugin ended up installing under `/Program Files (x86)/obs-studio` .

My previous OBS instance has been installed under `/Program Files (x86)/`. 

As a result VirtualCam didn't show up in Tools menu. Perhaps VirtualCam installer found this destination somewhere among registry entries (at some point of time a had versions of OBS Studio installed, while actually I hoped the new version would replace the older one).

What worked for me:
- uninstall OBS Studio
- install OBS Studio targeting the installer at `/Program Files (x86)/`

Not sure if this is a bug or just a corner case with multiple Studio installations available on the system at the moment VirtualCam gets installed.

**Comment by E3V3A** [comment]
Yep i can confirm this is an issue. In the installer, you can only chose one, not both, and if you attempt to re-run the installer, it remembers what it did before, and doesn't give an option to select another location.
